models/Product.js

```diff
@@ -0,0 +1,54 @@
+// import important parts of sequelize library
+const { Model, DataTypes } = require('sequelize');
+// import our database connection from config.js
+const sequelize = require('../config/connection');
+
+// Initialize Product model (table) by extending off Sequelize's Model class
+class Product extends Model { }
+
+// set up fields and rules for Product model
+Product.init(
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    product_name: {
+      type: DataTypes.STRING(120),
+      allowNull: false,
+    },
+    price: {
+      type: DataTypes.DECIMAL,
+      allowNull: false,
+      validator: {
+        isDecimal: true
+      }
+    },
+    stock: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      defaultValue: 10,
+      validator: {
+        isNumeric: true
+      }
+    },
+    category_id: {
+      type: DataTypes.INTEGER,
+      references: {
+        model: 'category',
+        key: 'id',
+      },
+    }
```

Comment:
Just as an FYI, Sequelize also offers onDelete and onUpdate options so that if you ever wanted a product to be deleted/set to null/not changed if its category is deleted you could specify that here.

You can find info on the different options here: https://sequelize.org/v5/class/lib/model.js~Model.html